antora-playbook.yml

```diff
@@ -22,7 +22,6 @@ site:
         { "title": "Autonomous Operator", "components": ["operator"] },
         { "title": "CMOS", "components": ["cmos"] },
         { "title": "SDKs", "startPage": "home::sdk.adoc", "components": ["*-sdk", "cxx-txns", "elasticsearch-connector", "kafka-connector", "spark-connector", "tableau-connector", "power-bi-connector", "sdk-extensions"] },
```

Comment:
I think you need to remove the trailing `,` here.

(Though this is in yaml, it's actually a *string* representation of JSON for, er... historical reasons, I imagine...)
And I think JSON spec strictly speaking doesn't allow trailing commas, even though JS itself does, 🤷 hey ho.